src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/Borough.java

```diff
@@ -71,4 +83,9 @@
     @JoinColumn(name = "BOROUGH_ID")
     private List<District> districts;
 
+    @ManyToMany
+    @JoinTable(name = "R_LEVEL_2_HEAD_OF_LEVEL_2",
+        joinColumns = @JoinColumn(name = "BOROUGH_ID"),
```

Comment:
Is R_LEVEL_2_HEAD_OF_LEVEL_2 a database view containing staff at the grade that are PDU heads? i.e. PSA, PSB, PSC.

Reply:
The table was added recently in Delius to allow authorised users to explicitly mark who are the heads of the PDU, regardless of staff grade.  Among other reasons, I believe this was so staff could be temporarily added while people were on leave etc - without having to change people's grades. We originally went down the route of looking up by PSB grade, until we discovered this Delius change which seems more appropriate.

It's worth noting however that this change is not currently live.  It's in all the lower environments and is planned to go-live next weekend (26th March).  As part of the go-live, they'll be running a script to populate the table based on information directly from the PDUs, so it should be accurate.  They've already tested this in Pre-Prod, so feel free to check that the data in there matches what you're after.

Reply:
Thanks Marcus. All good then.